jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/Jackson2Annotator.java

```diff
@@ -45,9 +46,11 @@
  *      href="https://github.com/FasterXML/jackson-annotations">https://github.com/FasterXML/jackson-annotations</a>
  */
 public class Jackson2Annotator extends AbstractAnnotator {
-
+    private GenerationConfig generationConfig;
```

Comment:
This class doesn't need this field, it should call `getGenerationConfig` from the parent.